server/src/services/testimonial.ts

```
@@ -0,0 +1,110 @@
+          status: testimonial.status,
+          comment: testimonial.comment
+        })
+        .from(testimonial)
+        .where(eq(testimonial.profile_id, profileId))
+        .orderBy(desc(testimonial.created_at));
+
+      resolve(testimonials);
+    } catch (error) {
+      reject(error);
+    }
+  });
```

Biome (2.1.2)

[error] Promise executor functions should not be `async`.



(lint/suspicious/noAsyncPromiseExecutor)